doc/reference/API_Reference.md

```diff
@@ -463,6 +469,9 @@ properties, methods, events and so on.
     Returns information on the key system currently attached to the
     HTMLMediaElement linked to the RxPlayer.
 
+  - [`getLastStoredContentPosition`](../api/Content_Information/getLastStoredContentPosition.md):
+    Returns the last stored content position, in seconds. 
+
```

Comment:
:O You didn't forget about the reference, I was unsure that I told you what it was. Nice!